generators/server/templates/quarkus/src/main/resources/application.properties.ejs

```diff
@@ -98,6 +98,8 @@ quarkus.oidc.authentication.restore-path-after-redirect=false
 quarkus.oidc.roles.role-claim-path=roles
 quarkus.oidc.roles.source=userinfo
 quarkus.oidc.authentication.user-info-required=true
+quarkus.oidc.token.refresh-expired=true
+quarkus.oidc.token.refresh-token-time-skew=10S
```

Comment:
Just curious: why is this needed?

Reply:
from docs:
<img width="1314" alt="image" src="https://user-images.githubusercontent.com/15997681/218273132-4123690b-55ca-480a-8418-7f33bbf53afe.png">

Reply:
Thanks! Automatically refreshing tokens is a cool feature. 😎